### PR TITLE
Genesis G70 fingerprint and tuning

### PR DESCRIPTION
**Pull request by ohber**
**Description**
- Update values.py to match entries for Genesis G70 from xps fork (stock does not work without these changes)
- Update tuning values in interface.py
- Add G70 to car list for 384 torque

**Verification**
- 384 torque and values.py changes have been present in xps' fork, and I have encountered no issues (used for 2+ months)
- Tuning is slightly changed from xps' fork but verified working well on various roads and speeds

**Route**
70c5bec28ec8e345|2020-10-25--11-42-28

### CODE REVIEW

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -275,9 +275,9 @@ class Buttons:
   "use_elect_gears": set([CAR.KIA_OPTIMA_H, CAR.IONIQ_EV_LTD, CAR.KONA_EV, CAR.IONIQ]),
 
   # these cars use the FCA11 message for the AEB and FCW signals, all others use SCC12
-  "use_fca": set([CAR.SONATA, CAR.ELANTRA, CAR.ELANTRA_GT_I30, CAR.KIA_STINGER, CAR.IONIQ, CAR.KONA_EV, CAR.KIA_FORTE, CAR.PALISADE, CAR.GENESIS_G70, CAR.KONA]),
+  "use_fca": set([CAR.SONATA, CAR.ELANTRA, CAR.ELANTRA_GT_I30, CAR.KIA_STINGER, CAR.IONIQ, CAR.KONA_EV, CAR.KIA_FORTE, CAR.PALISADE, CAR.KONA]),
 
-  "use_bsm": set([CAR.SONATA, CAR.PALISADE, CAR.HYUNDAI_GENESIS, CAR.GENESIS_G70, CAR.GENESIS_G80, CAR.GENESIS_G90, CAR.KONA]),
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/values.py:280]: are you sure about this? our car unit tests should have caught this if it was an issue

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -151,7 +151,7 @@ class Buttons:
 }
 
 # Don't use these fingerprints for fingerprinting, they are still used for ECU detection
-IGNORED_FINGERPRINTS = [CAR.VELOSTER, CAR.GENESIS_G70, CAR.KONA]
+IGNORED_FINGERPRINTS = [CAR.VELOSTER, CAR.KONA]
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/values.py:154]: revert this change and add your missing FW versions

**Comment by adeebshihadeh** [comment]
Can you rebase to fix the merge conflict?

File: selfdrive/car/hyundai/values.py (hunk 1 above)
Comment (ohber) [reply selfdrive/car/hyundai/values.py:280]: This change was made to match what xps had in his fork, which I've been on since I got my C2. Otherwise OP says my car is unsupported. I'm fairly confident that I'm currently the only user with this car model

Comment (adeebshihadeh) [reply selfdrive/car/hyundai/values.py:280]: this wouldn't result in not detecting your car, can you revert and try it?

**Comment by ohber** [comment]
I've verified that the fingerprints currently in values.py match exactly what my car is producing; removing the car from 'ignored fingerprints' appears to be a necessary change.

**Comment by ohber** [comment]
Realized I broke my branch when trying to rebase - everything should be correct now
